packages/node/src/defaults.js

```diff
@@ -22,6 +22,8 @@ const defaultPaths = getDefaultPaths('node')
  * @property {string} [projectRoot]
  * @property {boolean} [debugMode]
  * @property {boolean} [statsMode]
+ * @property {string} [policy] - Alias for `policyPath`?
+ * @property {string} [policyOverride] - Alias for `policyOverridePath`?
```

Comment:
worth trying to clarify what these two are before merge?
(the removing the question marks)

Reply:
these two props were only referenced in [prepareScenarioOnDisk](https://github.com/LavaMoat/LavaMoat/blob/8d866f3af42c79593fcf2201ef66d5b1b5855851/packages/core/test/util.js#L490), but it appears they were not actually used by any scenario.

I've removed them and changed the references to the correct `policyPath` and `policyOverridePath`.